tools/db_crashtest.py

```diff
@@ -855,6 +855,15 @@ def exit_if_stderr_has_errors(stderr, print_stderr=True):
         print("TEST FAILED. Output has 'fail'!!!\n")
         sys.exit(2)
 
+def cleanup_and_exit_if_failed(dbname):
```

Comment:
This function sounds like it's for when the crash test fails. It's for the opposite. How about `cleanup_after_success`? This is a shell script style python script, so exiting on fatal error is not unusual IMHO

Reply:
Thanks for the suggestion! I actually didn't like the name that I put here. I was thinking something like `cleanup_and_exit_if_cleanup_failed`, too but that was still meh.

I will change this to `cleanup_after_success`